onecodex/models/collection.py

```diff
@@ -317,24 +315,18 @@ def _classification_fetch(self, skip_missing=None):
                 "SampleCollection contains multiple analysis types: {}".format(", ".join(job_names))
             )
 
-        self._cached["is_metagenomic"] = False
-        if len(job_names) == 1 and "One Codex Database" in list(job_names)[0]:
-            self._cached["is_metagenomic"] = True
-
-        self._cached["classifications"] = new_classifications
+        return new_classifications
 
     @property
     def _classifications(self):
-        if "classifications" not in self._cached:
-            self._classification_fetch()
-
-        return self._cached["classifications"]
+        return self._classification_fetch()
```

Comment:
nit: `_classification_fetch()` isn't really *fetching* anything -- can we inline `_classification_fetch()` code in this `_classifications()` property?